src/components/application_manager/src/policies/policy_handler.cc

```diff
@@ -1765,11 +1765,11 @@ void PolicyHandler::OnCertificateDecrypted(bool is_succeeded) {
 void PolicyHandler::OnCertificateUpdated(const std::string& certificate_data) {
   LOG4CXX_AUTO_TRACE(logger_);
   sync_primitives::AutoLock lock(listeners_lock_);
-  HandlersCollection::const_iterator it = listeners_.begin();
-  for (; it != listeners_.end(); ++it) {
-    PolicyHandlerObserver* observer = *it;
-    observer->OnCertificateUpdated(certificate_data);
-  }
+  std::for_each(
```

Comment:
Do these changes pertain to this feature... they look to be general housekeeping?